website/scripts/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+MMVERSION=${1-0.3.35}
+
+echo Running website in container...
```

Comment:
Quote echo strings for consistency with other scripts.

Reply:
+1, this makes it harder to read without the quotes